automatic/googlechromebeta/googlechromebeta.nuspec

```diff
@@ -3,7 +3,7 @@
 <package xmlns="http://schemas.microsoft.com/packaging/2011/08/nuspec.xsd">
   <metadata>
     <id>googlechromebeta</id>
-    <version>73.0.3683.56-beta</version>
+    <version>73.0.3683.67-beta</version>
```

Comment:
If this is the latest version available of the google chrome beta, the package won't ever be updated.

Please change the version number to something lower (or change it to `0.0`).